cmd/envbuilder/main_test.go

```diff
@@ -1,84 +0,0 @@
-package main
-
-import (
-	"context"
-	"encoding/json"
-	"net/http"
-	"net/http/httptest"
-	"path/filepath"
-	"testing"
-	"time"
-
-	"cdr.dev/slog/sloggers/slogtest"
-	"github.com/coder/envbuilder/internal/notcodersdk"
-	"github.com/coder/serpent"
-	"github.com/google/uuid"
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
-)
-
-func Test_sendLogs(t *testing.T) {
```

Comment:
Do we have an integration test for this?

Reply:
No, not yet. I think this could be a good candidate for a follow-up, but it would mean we need to:
- Stand up an in-memory Coder instance
- Create an envbuilder template (we currently keep envbuilder templates in coder/coder)
- Assert the logs get sent to Coder

It would be fairly heavyweight all in all. What I'd really like is a more lightweight `fluentd`-like utility that just consumes stdin and forwards to a Coder instance.